fpgaconvnet/models/layers/ConvolutionLayer.py

```diff
@@ -165,13 +176,13 @@ def __init__(
                 self.modules["vector_dot"] = SparseVectorDot(self.rows_out(), self.cols_out(),
                     self.channels_in()//(self.coarse_in*self.coarse_group),
                     self.filters//(self.coarse_out*self.groups),
-                    self.kernel_size, self.sparsity, self.fine,
+                    self.kernel_size, self.sparsity, self.window_sparsity, self.skipping_windows, self.fine,
                     backend=self.backend, regression_model=self.regression_model)
 
                 self.modules["accum"] = Accum(self.rows_out(), self.cols_out(),
```

Comment:
We should also have a `SparseAccum` module, as that has different hardware as well